core/modules/layout/includes/layout.class.inc

```diff
@@ -250,6 +250,11 @@ class Layout {
         $this->{$property_key}[$plugin_type_key] = layout_create_handler($plugin_type, $plugin_data['plugin'], $plugin_data['data']);
       }
     }
+
+    // Ensure contexts is set to an array, even if empty.
+    if (is_null($this->contexts)) {
```

Comment:
Should this better be:

```suggestion
    if (!is_array($this->contexts)) {
```

Reply:
What was it on your site @klonos? On mine it was NULL.

Reply:
It happened on a PR sandbox, while I was testing https://github.com/backdrop/backdrop/pull/3274